packages/components/example-button/package-lock.json

```diff
@@ -25,7 +25,7 @@
 		},
 		"core-js": {
 			"version": "1.2.7",
-			"resolved": "https://registry.npmjs.org/core-js/-/core-js-1.2.7.tgz",
+			"resolved": "http://registry.npmjs.org/core-js/-/core-js-1.2.7.tgz",
```

Comment:
It doesn't feel like this file should have changed?

Reply:
Don't get me started 😂 

Hopefully after https://github.com/BBC-News/psammead/pull/190 is merged I'll update this PR and everything will be fine.